Could not find method google() for arguments [] on repository container

**Describe the bug**
Last version of the plugin causes an error when building the project in ionic.

**To Reproduce**
Create a blank ionic (v3) project.
`ionic start cordova-firebase-plugin blank`
Add the plugin
```
ionic cordova plugin add cordova-plugin-firebase
npm i @ionic-native/firebase
```
Build the app
```
ionic cordova build android --prod
```

**Expected behavior**
The project is build with no errors

**Console Logs**
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/yc00033/Documents/temp/cordova-firebase-plugin/platforms/android/build.gradle' line: 25

* What went wrong:
A problem occurred evaluating root project 'android'.

> Could not find method google() for arguments [] on repository container.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.653 secs
Error: /Users/yc00033/Documents/temp/cordova-firebase-plugin/platforms/android/gradlew: Command failed with exit code 1 Error output:
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/yc00033/Documents/temp/cordova-firebase-plugin/platforms/android/build.gradle' line: 25

* What went wrong:
A problem occurred evaluating root project 'android'.
> Could not find method google() for arguments [] on repository container.
```

**Plugin Version**
cordova-plugin-firebase 2.0.2 "Google Firebase Plugin"

**Additional context**
I created a project to replicate the issue.
[](https://github.com/davidanaya/cordova-firebase-plugin)

--- a/scripts/lib/android-helper.js
+++ b/scripts/lib/android-helper.js
@@ -21,18 +21,18 @@ function addDependencies(buildGradle) {
   // find the known line to match
   var match = buildGradle.match(/^(\s*)classpath 'com.android.tools.build(.*)/m);
   var whitespace = match[1];
-  
+
   // modify the line to add the necessary dependencies
   var googlePlayDependency = whitespace + 'classpath \'com.google.gms:google-services:4.1.0\' // google-services dependency from cordova-plugin-firebase';
   var fabricDependency = whitespace + 'classpath \'io.fabric.tools:gradle:1.25.4\' // fabric dependency from cordova-plugin-firebase'
   var modifiedLine = match[0] + '\n' + googlePlayDependency + '\n' + fabricDependency;
-  
+
   // modify the actual line
   return buildGradle.replace(/^(\s*)classpath 'com.android.tools.build(.*)/m, modifiedLine);
 }
 
 /*
- * Add 'google()' to the repository repo list
+ * Add Crashlytics repos to the repository list
  */
 function addRepos(buildGradle) {
   // find the known line to match
@@ -40,9 +40,8 @@ function addRepos(buildGradle) {
   var whitespace = match[1];
 
   // modify the line to add the necessary repo
-  var googlesMavenRepo = whitespace + 'google() // Google\'s Maven repository from cordova-plugin-firebase';
   var fabricMavenRepo = whitespace + 'maven { url \'https://maven.fabric.io/public\' } // Fabrics Maven repository from cordova-plugin-firebase'
-  var modifiedLine = match[0] + '\n' + googlesMavenRepo + '\n' + fabricMavenRepo;
+  var modifiedLine = match[0] + '\n' + fabricMavenRepo;
 
   // modify the actual line
   return buildGradle.replace(/^(\s*)jcenter\(\)/m, modifiedLine);
@@ -68,7 +67,7 @@ module.exports = {
 
     // Add Google Play Services Dependency
     buildGradle = addDependencies(buildGradle);
-  
+
     // Add Google's Maven Repo
     buildGradle = addRepos(buildGradle);
 
@@ -85,7 +84,7 @@ module.exports = {
 
     // remove any lines we added
     buildGradle = buildGradle.replace(/(?:^|\r?\n)(.*)cordova-plugin-firebase*?(?=$|\r?\n)/g, '');
-  
+
     writeRootBuildGradle(buildGradle);
   }
 };

